main.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_s3_bucket" "demo-s3" {
   bucket = "${var.s3_bucket_prefix}-terraformcloud"
   versioning {
     enabled = false
-    mfa_delete = false
+    mfa_delete = true
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should have versioning enabled**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites
 
**Remediation**
S3 bucket should have versioning enabled

source: accurics-